deploy/kubernetes/controller.yaml

```diff
@@ -62,7 +62,7 @@ spec:
               mountPath: /csi
         - name: packet-driver
           imagePullPolicy: Always
-          image: gcr.io/stackpoint-public/csi-packet-driver:v0.1.0
+          image: docker.io/surajd/csi-packet:latest-amd64
```

Comment:
This is temporary since I wanted to try out my changes.

Reply:
When this is ready, we can point at `docker.io/packethost/csi-packet`

Reply:
done